src/main/host/syscall_numbers.h

```diff
@@ -10,7 +10,7 @@
 
 typedef enum {
     SYS_shadow_min = 1000,
-    SYS_shadow_set_ptrace_allow_native_syscalls = 1000,
+    // Deprecated: SYS_shadow_set_ptrace_allow_native_syscalls = 1000,
```

Comment:
It's not used anywhere, and neither are the other commented-out variants. I'm not sure why the rest are commented-out rather than removed, but I just did the same here. My guess is maybe they were left so that it's easier to look through old log messages and know what those syscalls were, and so that we don't reuse the syscall number in the future in which case the meaning of that syscall number would change between old logs and new logs.

I'll add a comment with my guess above, but if you would rather me just remove them instead, let me know.

Reply:
Or if @sporksmith remembers, let me know and I can add a comment.

Reply:
Yup, that's right.

Avoiding reusing syscall numbers would make it a bit less confusing if we somehow ended up running using an old version of the shim that tries to make those syscalls, or some custom program that made such syscalls itself e.g. to opt out of interposition.

The chances of those happening are low, but OTOH it doesn't cost us anything to keep the comments around. I'd lean towards keeping them, but would be fine with removing them if others prefer it.

Reply:
Forgot to mention - and yeah, keeping the comments with the mappings of the deprecated syscalls would also make it a little easier to debug what's happening in the case where we *did* end up seeing something try to use the deprecated syscalls.

Reply:
Thanks for the clarifications! I agree that keeping them around does seem like a good idea :)

Reply:
Thanks! I've added some comments.